AxeWP/wp-graphql-rank-math
# fix: ensure required composer deps are in repo.

## What

Ensures the `axepress` dependency is kept in the plugin repo.
## Why
Closes #1 

## How

## Testing Instructions
1. Download .zip of branch.
2. Install and activate.
3. *Don't* see fatal error.

## Additional Info

- In the future, all deps should be removed from the repo and a GH action should instead generate a release artifact.

## Checklist:
